src/FakeItEasy/Core/FakeManager.PropertyBehaviorRule.cs

```diff
@@ -39,25 +39,18 @@ public static bool IsPropertyGetter(MethodInfo method)
                 return method.IsSpecialName && method.Name.StartsWith("get_", StringComparison.Ordinal);
             }
 
+            public bool IsMatchForSetter(IFakeObjectCall fakeObjectCall) => IsPropertySetter(fakeObjectCall);

```

Comment:
If you have a better name to suggest, I'll gladly take it. I didn't think too long about this one.